scripts/pre_commit_linter.py

```diff
@@ -463,8 +470,238 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
+def _get_all_files():
+    """This function is used to parse the arguments entered by the user, and
+    returns the list of all files for linting and pattern checks.
+
+    Returns:
+        tuple(list(str), bool). The tuple consisting list of files for linting
+        and pattern checks and bool which checks if whether verbose
+        mode is enabled or not.
+    """
+    eslintignore_path = os.path.join(os.getcwd(), '.eslintignore')
+    parsed_args = _PARSER.parse_args()
+    # Default mode is non-verbose mode, if arguments contains --verbose flag it
+    # will be made True, which will represent verbose mode.
+    verbose_mode_enabled = bool(parsed_args.verbose)
+    if parsed_args.path:
```

Comment:
I suggest you pass `parsed_args.path` and `parsed_args.files` into this function as separate args. This function shouldn't need to do argparsing.

Reply:
Done

Reply:
Hi @seanlip, I have named the parameters of _get_all_files as 'input_files' & 'input_path'  as there beacuse I named earlier 'files', and 'path' but path was being used globally so pylint was showing error message ```redefined-outer-name```.